js/test/components/operatorSignIn/attestation.test.tsx

```diff
@@ -97,17 +136,32 @@ describe("Attestation", () => {
   });
 
   test("valid attestation", async () => {
+    // 'Radio' input state is held a component level up to keep
+    //   retry easy. Therefore we're pushing "22" in as a prop,
```

Comment:
I wonder if it's possible to set this up more realistically? Something like
```tsx
const Wrapper = () => {
  [radio, setRadio] = useState("");
  return <Attestation radio={radio} setRadio={setRadio} />
}
render(<Wrapper />);
```

Reply:
I changed this so Attestation submits the radio number via the `onComplete`, which the modal can then use to set its state as well as submit the sign-in. Seems better!